angular.json

```diff
@@ -3,7 +3,7 @@
   "version": 1,
   "newProjectRoot": "projects",
   "projects": {
-    "intro-storybook-angular-template": {
+    "jazz-design-system": {
```

Comment:
I have been deliberately leaving "jazz" out of package and folder names because marketing terms are the most likely to change.  In fact, the marketing name has already changed more than once.

We should stick with just "design-system".  The NPM scope will distinguish it from other design systems.